### PR TITLE
[WIP] Github actions in branch in this repo

### PR DESCRIPTION
**Pull request by robintibor**
Hm does it work this way in PR in same repo?

### CODE REVIEW

File: docs/requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ sphinx_rtd_theme
 numpydoc
 memory_profiler
 pillow
+ipython
```
Comment (agramfort) [inline docs/requirements.txt:6]: I would have rather killed the `from IPython.display import display` from the examples :)

**Comment by robintibor** [comment]
Were you aware of this https://github.com/actions/upload-artifact/issues/14 @agramfort ? So basically the HTML Documentation Artifact will be a single zip file that you have to download, can't just click through it, e.g., see https://github.com/braindecode/braindecode/actions/runs/742481627

**Comment by agramfort** [comment]
yes I've used it before. It's not as convenient as circle-ci though as you
don't have a rendered version online in one click. You
need to download and you cannot point people to one page with an HTML link

>

**Comment by robintibor** [comment]
But plan would still be to use it to replace both CircleCI and Travis no? otherwise so far seems to me could turn out to be less complex than before, so I like that part.

**Comment by agramfort** [comment]
shall we merge this after removing travis? you'll need to update badges on the readme file.

**Comment by robintibor** [comment]
well right now it also builds docs, if we stick with CircleCI would we still want it to build docs? as like second check to compare to CircleCI behavior maybe not even so stupid?

**Comment by robintibor** [comment]
@agramfort I suggest let's first merge https://github.com/braindecode/braindecode/pull/190 then rebase this one? Then no need to rename the plot_relative_positioning and plot_sleep_staging.py at least if one is willing to wait the 30min

**Review by agramfort — APPROVED** [review]
@robintibor feel free to merge

**Comment by robintibor** [comment]
But don't we want to revent/undo the renamings e.g. examples/plot_relative_positioning.py → examples/relative_positioning.py ?

**Comment by agramfort** [comment]
let's get the CIs green and can put it back in a next PR. One example at a
time

>

**Pull request merged**